recipes/nspr/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "4.19" %}
+
+package:
+  name: nspr
+  version: {{ version }}
+
+source:
+  fn: nspr-{{ version }}.tgz
+  url: https://ftp.mozilla.org/pub/nspr/releases/v{{ version }}/src/nspr-{{ version }}.tar.gz
+  sha256: 2ed95917fa2277910d1d1cf36030607dccc0ba522bba08e2af13c113dcd8f729
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - toolchain
+    - make
+    - autoconf
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libnspr4.a         # [unix]
+    - test -f ${PREFIX}/lib/libnspr4.a         # [unix]
+    - test -f ${PREFIX}/lib/libnspr4.so        # [linux]
+    - test -f ${PREFIX}/lib/libplc4.a          # [unix]
+    - test -f ${PREFIX}/lib/libplc4.so         # [linux]
+    - test -f ${PREFIX}/lib/libplc4.dylib      # [osx]
+    - test -f ${PREFIX}/lib/libplds4.a         # [unix]
+    - test -f ${PREFIX}/lib/libplds4.so        # [linux]
+    - test -f ${PREFIX}/lib/libplds4.dylib     # [osx]
```

Comment:
We can replace the `.dylib` and `.so` with `${SHLIB_EXT}`. Then these can all be `unix`.

Reply:
Good to know! Ty!